### PR TITLE
add dinner menu

### PR DESCRIPTION
**Pull request by Supakrit65**
I have added dinner menu and my about us info.

### CODE REVIEW

**Comment by Halcyon905** [comment]
forgot to make a hyperlink

**Comment by RyukungG** [comment]
You haven't make a link to your section in README.md yet.
You title is wrong you title now is Breakfast

File: README.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
 You learned the Markdown to do this in the `git-commands` assignments.    
 **TODO**: Remove all the TODO lines, of course.
 
+## Dinner
+| Menu          | Cost (THB) |
+|:--------------|-----------:|
+| Oatmeal       |      40.00 |
+| omelette      |      20.00 |
+| Mushroom Soup |      60.00 |
+| Milkshake     |      30.00 |
```
Comment (GToidZ) [inline README.md]: Sort alphabetically, "omelette" should start with capital letter.

**Review by GToidZ — CHANGES_REQUESTED** [review]
Looks good, but a little more tweak.

**Review by Halcyon905 — APPROVED** [review]
Looks good!

**Review by GToidZ — APPROVED** [review]
Looks good. Merge approved.

**Pull request merged**